OasisDEX/oasis-borrow
# Stop loss maker metadata

# [Stop loss maker metadata](https://app.shortcut.com/oazo-apps/story/6698/metadata-for-automation)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- moved all maker-specific validations, calculations, copies to metadata config
  
## How to test 🧪
  <Please explain how to test your changes>

- stop loss should work as usual, please test add / update / remove trigger flows, pay attention to copies, modal content etc.
